specs/MultiProfile.md

```diff
@@ -363,16 +384,36 @@ interface ICoreWebView2Profile2 : ICoreWebView2Profile {
   [propget] HRESULT CookieManager([out, retval] ICoreWebView2CookieManager** cookieManager);
 }
 
-[uuid(1c1ae2cc-d5c2-ffe3-d3e7-7857035d23b7), object, pointer_default(unique)]
-interface ICoreWebView2Profile3 : ICoreWebView2Profile2 {
-  /// All webviews on this profile will be closed, and the profile will be marked for deletion.
-  /// After the Delete() call completes, The render process of webviews on this profile will
-  /// asynchronously exit with the reason:`COREWEBVIEW2_PROCESS_FAILED_REASON_PROFILE_DELETED`.
-  /// See 'COREWEBVIEW2_PROCESS_FAILED_REASON::COREWEBVIEW2_PROCESS_FAILED_REASON_PROFILE_DELETED'
-  /// for more details. The profile directory on disk will be actually deleted when the browser
-  /// process exits. Webview2 creation will fail with the HRESULT is ERROR_INVALID_STATE(0x8007139FL)
-  /// if you create it with the same name as a profile that is being deleted.
-  HRESULT Delete(); 
+[uuid(2765B8BD-7C57-4B76-B8AA-1EC940FE92CC), object, pointer_default(unique)]
+interface ICoreWebView2StagingProfile4 : IUnknown {
+  /// After the API is called, the profile will be marked for deletion. The
+  /// local profile’s directory will be tried to delete at browser process
+  /// exit, if fail to delete, it will recursively try to delete at next
+  /// browser process start until successful.
+  /// The corresponding user's `ProfileDeleted` event handle function will
+  /// be triggered. After the function is triggered, continuing to use the
+  /// profile or its corresponding webviews is an undefined behavior.
```

Comment:
We should define the behavior not document it as undefined. Right now, if CoreWebView2Controller.Close is called, or if the webview2 browser process crashes (a few other cases as well), the corresponding CoreWebView2 objects are put into a closed state and we document what happens when methods are called or properties are accessed when in the closed state (IIRC we throw an exception / return a FAILED(HRESULT)). We should probably do the same here - put the objects into a closed state and link to the documentation that describes what happens when you try to use an object in its closed state.

Reply:
Some can work normally, but some of them can't work properly, such as async API cannot be successful the await will never return, for the webview will be auto closed. 
So, by the Option1 we do not allow user to continue to use the webview although some sync API can still be used.
And we can change the comments to "After the function is triggered, cannot continue to use the profile and its corresponding webviews."

Reply:
We can discuss in the other issue thread about this below. If we don't want end devs to use the corewebview2 or profile objects we should close them before raising the event.